src/applications/vaos/tests/appointment-list/components/ConfirmedAppointmentDetailsPage/index.video.unit.spec.jsx

```diff
@@ -1298,23 +1238,18 @@ describe('VAOS <ConfirmedAppointmentDetailsPage>', () => {
         },
       };
 
-      mockAppointmentInfo({
-        va: [appointment],
-        cc: [],
-        requests: [],
+      mockSingleAppointmentFetch({
```

Comment:
Same as above, I opted to simplify this test and remove the list -> details transition